src/app/component/user/modal-auth/windowsignin/windowsignin.component.css

```diff
@@ -0,0 +1,223 @@
+    display: flex;
+    justify-content: space-between;
+    max-width: 800px;
+    position: relative;
+}
+
+.sign-in {
+    width: 100%;
+    height: auto;
+}
+
+label[for="email"],
+label[for="password"],
+label[for="first-name"] {
+    font: 14px 'OpenSans', sans-serif;
+    letter-spacing: 0.34px;
+    color: #839c94;
+    margin-top: 10px;
+}
+
+form>h1 {
+    font: 30px 'OpenSans-Bold', sans-serif;
+    letter-spacing: 0.18px;
+    font-weight: 400;
+    margin: 10% 0;
+}
+
+form>h2 {
+    font: 16px 'OpenSans-Regular', sans-serif;
+    line-height: 22px;
+    letter-spacing: 0.18px;
+    font-weight: 500;
+    margin-top: 5px;
+}
+
+.width-inp {
+    width: 100%;
+}
+
+.close-icon {
+    position: absolute;
+    right: 10px;
+}
+
+.sign-in-input {
+    display: block;
+    height: 35px;
+    border-radius: 5px;
+    -webkit-border-radius: 5px;
+    -moz-border-radius: 5px;
+    -o-border-radius: 5px;
+    border: 1px solid #839c94;
+    background-color: rgba(131, 156, 148, 0.07);
+    padding: 0 10px;
+    width: 100%;
+}
+
+.form-pic {
+    width: 100%;
+}
+
+.form-pic>img {
+    width: 100%;
+    height: auto;
+    margin-left: -24px;
+    margin-top: -24px;
+    margin-bottom: -24px;
+}
+
+.error,
+.validation-error {
+    color: #f03127;
+    font-size: 12px;
+}
+
+form {
+    margin: 2% 5% 5% 5%;
+}
+
+.red {
+    border: 1px solid #f03127 !important;
+    background: #FEF2F1;
+}
+
+.grey {
+    border: 1px solid #839c94;
+}
+
+.password-wrapper {
+    position: relative;
+    display: flex;
+}
+
+.submit {
+    width: 100%;
+    height: 35px;
+    background: #13aa57;
+    color: #ffffff;
+    border: none;
+    border-radius: 4px;
+    -webkit-border-radius: 4px;
+    -moz-border-radius: 4px;
+    -o-border-radius: 4px;
+    margin-top: 15px;
+    cursor: pointer;
+}
+
+.img {
+    display: inline;
+    z-index: 2;
+    cursor: pointer;
+    width: 24px;
+    height: 20px;
+    margin-top: 7px;
+    margin-right: 0px;
+    margin-left: -24px;
+    background-image: url('src/assets/img/icon/eye.png');
+}
+
+.forgot-wrapper {
+    display: flex;
+    justify-content: flex-end;
+}
+
+a.forgot-password {
+    color: #13aa57;
+    font-weight: 500;
+    margin-top: 15px;
+}
+
+.wrapper-sign-up {
+    margin: 0 5%;
+    text-align: center;
+}
+
+.haveyou-account {
+    margin-top: 5%;
+    padding-left: 5%;
+}
+
+.signup-link {
+    color: #13aa57;
+    font-weight: 500;
+    cursor: pointer;
+    text-decoration: underline;
+}
+
+.or {
+    color: #839c94;
+    margin-top: -5px;
+}
+
+.google {
+    border-radius: 4px;
+    -webkit-border-radius: 4px;
+    -moz-border-radius: 4px;
+    -o-border-radius: 4px;
+    background: #ffffff;
+    margin-top: 5px;
+    width: 100%;
+    height: 35px;
+    display: flex;
+    justify-content: space-between;
+    box-shadow: 0 3px 8px 2px rgba(0, 0, 0, 0.12);
+    border: 1px solid #ffffff !important;
+    background: #ffffff !important;
+}
+
+.google-arrow {
+    margin-top: 10px;
+}
+

```

Comment:
Please delete extra empty line

Reply:
I remember about one empty line in each file (it is for webpack). Is it rule mandatory for all types of files except CSS?

Reply:
One empty line between each class in the css

Reply:
ready